src/controllers/room.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response } from 'express';
+import Room from '../models/Room';
+
+export async function createRoom(req: Request, res: Response) {
+    const { name, sections, schedule, closed } = req.body;
+    const room = new Room({
+        name,
+        sections,
+        schedule,
+        closed,
+    });
+    await room.save().catch((err) => {
+        return res.status(400).json({ err });
+    });
+    res.status(201).json({ room });
+}
+
+export async function queryRoom(req: Request, res: Response) {
+    const id = req.query.id;
+    if (id == null) {
```

Comment:
Loose equality makes this harder to read; it might not be immediately clear to people that `undefined == null`. Would it be better to swap this around, and simply use a `if (id) {}`?

Reply:
Done